src/SDKs/Azure.Core/data-plane/Azure.Core.Tests/Azure.Core.Tests.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
We are not going to have other planes for this component. Do we need to nest it in data-plane?

Reply:
I put it in the data-plane folder because the current infrastructure in that repo knows to build projects in that folder. @weshaggard will move this folder to the final destination when we re-arrange the repo and will get rid of the data-plane folder.

Reply:
Yes once we finalize the repo structure plan we will move things as necessary.